tests/console/zypper_lifecycle_toolchain.pm

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: test for 'zypper lifecycle' for toolchain module
+# Maintainer: Rodion Iafarov <riafarov@suse.com>
+# Tags: fate#32205
+
+use base "consoletest";
+use strict;
+use testapi;
+
+sub log_failed_package {
+    my ($package, $exp_date, $output) = @_;
+    record_info("toolchain gcc package has wrong lifecycle output", "For $package expected $exp_date as expiration date, lifecyecle output:\n $output");
+}
+
+sub run() {
+    my ($self) = @_;
+    $self->result('ok');
```

Comment:
you need result('ok') if there is no other assertion in the test - aka very rarely